### ISSUE TITLE
Is European Union CSV file Missing?

### ISSUE THREAD

**Issue by zizwiz** [owid/covid-19-data#330]
I see you have data for European Union as a whole area but I see it missing in the folder 
covid-19-data/public/data/vaccinations/country_data/
 Is it coming or should I write my code to ignore it?

**Comment by edomt** [comment]
Hi @zizwiz 
The `country_data` folder will only ever include the input data for each country. The aggregates that we add on top of that (currently World and European Union) will not appear in that folder.

**Comment by zizwiz** [comment]
I understand but you have England, Scotland, Wales and N. Ireland and then you have United Kingdom?

**Comment by edomt** [comment]
That's because we really take them as separate inputs from the official UK API.
Basically what's listed in https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/locations.csv are the inputs. The rest are aggregates that are calculated as the sum of inputs.

**Comment by adamredfern92** [comment]
@edomt just to piggy back off this - I'm getting a different number of vaccinations when I sum the 27 EU member states (5,701,282) to what is there for the European Union (5,124,473)... what countries are being included?

**Comment by edomt** [comment]
@adamredfern92 Have you checked the dates?

**Comment by adamredfern92** [comment]
Ah I see, EU is still on 17th, but some countries have data for 18th. Will you always leave it an extra day behind?

**Comment by edomt** [comment]
Yep, we don't calculate aggregates for the very last day, otherwise they tend to look very low when countries haven't all reported yet.

**Comment by zizwiz** [comment]
Thanks, keep up the good work very helpful for me.
